### DISCUSSION TITLE
Laravel Sail - no OS dependencies

### DISCUSSION THREAD

**Discussion by GlennM** [TiagoSilvaPereira/vemto-issues#439] [Q&A] [answered]
Hi,

I've read something similar in https://github.com/TiagoSilvaPereira/vemto-issues/issues/102
That issue is about Windows, I'm on a clean version of macOS without any of the PHP, Composer & NPM dependencies. (I'm having git though).

Laravel introduced Laravel Sail, which makes it possible to run Laravel via Docker without the need to install any of the dependencies on the OS and use them via Docker.

Would be great if Vemto does support these dependencies via Docker, instead of relying on them to be installed on macOS.
Not sure if that's possible (due to the architecture of Vemto 1.x). If not, can we have this in V2 and is there already an ETA for V2?

Thanks!

**Comment by TiagoSilvaPereira** [comment, marked as answer]
Hi @GlennM, it is already in the plans of Vemto v2. With Vemto v1 it is impossible due to the architecture, but I created Vemto v2 with this in mind.
